redash/query_runner/big_query.py

```diff
@@ -146,7 +160,7 @@ def _get_bigquery_service(self):
         http = httplib2.Http(timeout=settings.BIGQUERY_HTTP_TIMEOUT)
         http = creds.authorize(http)
 
-        return build("bigquery", "v2", http=http)
+        return build("bigquery", "v2", http=http, cache_discovery=False)
```

Comment:
What this do?

Reply:
it's a bug fix when this kind of error happened
“ImportError: file_cache is unavailable” when using Python client for Google service account file_cache

Reply:
This is a known issue in the google-api-python-client library: https://github.com/googleapis/google-api-python-client/issues/299#issuecomment-427118293 I would at least apply this fix.